shell/src/main/java/org/apache/accumulo/shell/commands/CreateResourceGroupCommand.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.accumulo.shell.commands;
+
+import static org.apache.accumulo.shell.ShellUtil.readPropertiesFromFile;
+
+import java.util.Map;
+import java.util.Map.Entry;
+
+import org.apache.accumulo.core.client.AccumuloException;
+import org.apache.accumulo.core.client.AccumuloSecurityException;
+import org.apache.accumulo.core.client.ResourceGroupNotFoundException;
+import org.apache.accumulo.core.client.admin.ResourceGroupOperations;
+import org.apache.accumulo.core.data.ResourceGroupId;
+import org.apache.accumulo.shell.Shell;
+import org.apache.accumulo.shell.Shell.Command;
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.Option;
+import org.apache.commons.cli.Options;
+
+public class CreateResourceGroupCommand extends Command {
+
+  private Option createTableOptInitPropFile;
+
+  @Override
+  public int execute(String fullCommand, CommandLine cl, Shell shellState) throws Exception {
+    final String resourceGroup = cl.getArgs()[0];
+    final ResourceGroupId rgid = ResourceGroupId.of(resourceGroup);
+    final ResourceGroupOperations ops = shellState.getAccumuloClient().resourceGroupOperations();
+    try {
+      ops.create(rgid);
+    } catch (AccumuloException | AccumuloSecurityException e) {
+      Shell.log.error("Error creating resource group {}", resourceGroup, e);
+      return 1;
+    }
+
+    String filename = cl.getOptionValue(createTableOptInitPropFile.getOpt());
+    if (filename != null) {
+      final Map<String,String> initProperties = readPropertiesFromFile(filename);
+      for (Entry<String,String> e : initProperties.entrySet()) {
```

Comment:
If `ops.modifyProperties()` is used here instead of `ops.setProperty()` then can atomically set all the props at once and also will only make a one RPC instead of many RPCs.